default/scripting/common/priorities.macros

```diff
@@ -15,3 +15,5 @@ LATE_PRIORITY
 VERY_LATE_PRIORITY
 '''120'''
 
+LATEST_GROWTH_PRIORITY
+'''99999'''
```

Comment:
Was too hasty clicking approved, looks like other file has the added EOL but not this one (or browser is not updating)